app/common.cds

```diff
@@ -1,98 +1,464 @@
 /*
   Common Annotations shared by all apps
 */
-
-using { my.bookshop as my } from '../db/schema';
+using
```

Comment:
I've been using the Eclipse CDS file formatter... I'll use the one in VSCode with the default setting and reformat all CDS files that I changed... hopefully it will look fine, then.

Reply:
Is the Eclipse formatter different from the one in VS Code? I never saw this kind of formatting so far and was wondering about it therefore. It that's like the best practice format induced by the tools we can also keep it like this. For me personally this is taking up too much space, but formatting is a topic where you can have endless discussions..

Reply:
I think the options are the same in both as they're both based on the same language server. But somehow the defaults in Eclipse seem different. I find it easier to set it up correctly in VSCode.

Reply:
hm, that makes it nearly impractical to use out-of-the-box when working with multiple developers with different IDEs :(
I'm personally no friend of code formatters. In my feeling they never get it quite right, but thats - as said before - a topic of taste and endless discussion :)